src/main/java/com/karankumar/bookproject/ui/shelf/visibility/AllShelvesBookVisibility.java

```diff
@@ -0,0 +1,14 @@
+package com.karankumar.bookproject.ui.shelf.visibility;
```

Comment:
Could you add the [copyright notice](https://github.com/knjk04/book-project/blob/master/docs/COPYRIGHT) before the `package` statement, please?

Reply:
done